src/System.Net.Http/src/System/Net/Http/HttpClientHandler.cs

```diff
@@ -13,6 +13,7 @@ public partial class HttpClientHandler : HttpMessageHandler
     {
         // This partial implementation contains members common to all HttpClientHandler implementations.
         private const string ManagedHandlerSettingName = "COMPlus_UseManagedHttpClientHandler";
```

Comment:
Do we have an issue open to rename this?  Want to make sure we don't lose that.

Reply:
#26466 We can cleanup any outstanding (or new) concerns there. Environment variable is convenient but it seems like @tarekgh feels it is redundant. If anybody has suggestions please comment on new issue.